File: client_udp.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

using System;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Net;
using System.Net.Sockets;

public class Socket_01 : MonoBehaviour
{
    public void socket_client()
    {
        Socket client = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, ProtocolType.Udp);
        var data = Encoding.UTF8.GetBytes("Hello");
        client.SendTo(data, new IPEndPoint(IPAddress.Parse("192.168.0.11"), 43210));
        client.Close();
        Debug.Log(Encoding.UTF8.GetString(data));
    }
}

File: client_udp.py
import socket

HOST = "192.168.0.11"  # IPv4 Address
PORT = 43210           # Port Number, using dynamic ports : 49152 ~ 65535 

socket_client = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)

while True:
    data = input("Input Data : ")
    
    socket_client.sendto(data.encode(), (HOST,PORT))
    
socket_client.close()
